service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageQueries.kt

```diff
@@ -170,7 +170,8 @@ threads AS (
     WHERE EXISTS(
             SELECT 1
             FROM participated_messages rec
-            WHERE rec.thread_id = t.id AND rec.recipient_id = :accountId)
+            WHERE rec.thread_id = t.id
+            AND (rec.recipient_id = :accountId OR ${if (includeWhereSender) "rec.sender_id = :accountId" else "false"}))
```

Comment:
I'm not sure about this change. This may also break current employee side so that sent messages are shown in inbox

Reply:
Thanks for the feedback. Good point.

The parameter `includeWhereSender` was originally called something like `notEmployee`. It's intended to always be `false` when making an employee-side queries. This was later renamed as suggested by the other reviewer. Maybe `notEmployee` would indeed be more legible?

Reply:
Renamed the parameter, since in my opinion, it clarifies the context a bit.